tests/stis/test_lev1_fuvspec.py

```diff
@@ -0,0 +1,26 @@
+import subprocess
+from ci_watson.artifactory_helpers import get_bigdata
+
+from ..helpers import BaseSTIS
+
+
+class TestLev1FUVSpec(BaseSTIS):
+    """Level 1 FUV-MAMA spectroscopy."""
+    detector = 'fuv-mama'
+
+    def test_lev1(self):
+        """This was ``calstis_lev1_FUVspec``."""
+        raw_file = 'o5cl02040_raw.fits'
+        wav_file = 'o5cl02040_wav.fits'
+
+        # Prepare input files.
+        self.get_input_file(raw_file)
+        get_bigdata(self.env, 'stis', 'fuv-mama', 'input', wav_file)
+
+        # Run CALSTIS (equivalent to stistools.calstis.calstis)
+        subprocess.call(['cs0.e', raw_file, '-v'])
+
+        # Compare results
+        outputs = [('o5cl02040_flt.fits', 'calstis_lev1_FUVspec_flt.fits'),
+                   ('o5cl02040_x1d.fits', 'calstis_lev1_FUVspec_x1d.fits')]
+        self.compare_outputs(outputs)
```

Comment:
Note: This is a weird one, where if I run this same test on my own machine, the comparison would fail until I set `rtol=0.01` (1%). 🤷‍♀️  But it passes if run on Jenkins. Investigating this is out of scope of this PR.

Reply:
Attached is the failure I see locally, which Jenkins does not reproduce.

[stis_local_run.txt](https://github.com/spacetelescope/hstcal/files/2303410/stis_local_run.txt)